source/docs/software/advanced-control/state-space/state-space-flywheel-walkthrough.rst

```diff
@@ -1,4 +1,123 @@
-State-Space Controller Examples
-===============================
+State-Space Controller Walkthrough
+==================================
 
+.. note:: Before following this tutorial, readers are recommended to have read an :ref:`docs/software/advanced-control/state-space/state-space-intro:Introduction to state-space control`.
 
+The goal of this tutorial is to provide "end-to-end" instruction on implementing a state-space controller on a flywheel.  By following this tutorial, readers will learn how to:
+
+1. Create an accurate state-space model of a flywheel using :term:`system identification` or CAD software
+2. Implement a Kalman Filter to laglessly filter encoder velocity measurements.
```

Comment:
Replace with "2. Implement a Kalman Filter to filter encoder velocity measurements without lag."